AndroidNativeLoginTemplate/app/src/main/java/com/salesforce/androidnativelogintemplate/NativeLogin.kt

```diff
@@ -28,17 +28,22 @@ package com.salesforce.androidnativelogintemplate
 
 import android.app.Activity
 import android.content.Intent
-import android.os.Build
+import android.os.Build.VERSION.SDK_INT
+import android.os.Build.VERSION_CODES.S
+import android.os.Build.VERSION_CODES.TIRAMISU
 import android.os.Bundle
 import android.widget.Toast
-import android.window.OnBackInvokedDispatcher
+import android.widget.Toast.LENGTH_LONG
```

Comment:
I used static imports in places to improve readability and reduce some of the length namespaces where possible.